pkg/servicecentral/client_test.go

```diff
@@ -265,7 +265,90 @@ func TestGetService(t *testing.T) {
 
 	// then
 	require.NoError(t, err)
-	require.Equal(t, 1, handler.RequestSnapshots.Calls())
+	require.Equal(t, 2, handler.RequestSnapshots.Calls())
+}
+
+func TestGetServiceWithOpsGenieAttribute(t *testing.T) {
+	t.Parallel()
+	// given
+	handler := MockHandler(Match(
```

Comment:
nit: This doesn't jive with Go standards, and this indenting looks a bit confusing. Could you clean this up?

Reply:
Yeah no worries, interesting that our linting build didn't pick this up.